auto-pr-script/create_pull_requests

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail; [[ -n "$DEBUG" ]] && set -x
+
+if [[ $# -lt 5 ]]; then
+		echo "usage: $0 pull_request_id pull_request_url pull_request_title commit_sha assignee branches"
```

Comment:
This indent seems too deep?

Reply:
Indeed, thanks.

Reply:
Revised in https://github.com/scalar-labs/actions/pull/6/commits/6e0a4d635bc91ab8f7402178887322c40dac122c